pkg/test/test_data/crd-in-step/00-crd.yaml

```diff
@@ -3,6 +3,8 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: mycrds.mycrd.k8s.io
+  annotations:
+    api-approved.kubernetes.io: "https://github.com/kubernetes/enhancements/pull/1111"
```

Comment:
See the  [PR in the annotation](https://github.com/kubernetes/enhancements/pull/1111) for more details on why this is needed